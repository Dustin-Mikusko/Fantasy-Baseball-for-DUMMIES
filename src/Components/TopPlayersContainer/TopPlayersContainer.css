.top-container {
  display: flex;
  flex-direction: row;
  padding: 50px;
  width: 100%;
}

.stat-explanation {
  width: 40%;
  padding: 10px;
  font-size: 2em;
}

.top-player-container {
  width: 50%;
  justify-content: center;
}

.top-player-name {
  text-align: center;
  border-bottom: 1px solid #041F42;

}
