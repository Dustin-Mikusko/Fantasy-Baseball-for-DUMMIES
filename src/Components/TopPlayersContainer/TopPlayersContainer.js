import React from 'react';
import './TopPlayersContainer.css';
import TopPlayer from '../TopPlayer/TopPlayer'

export const TopPlayersContainer = ({ players, stat }) => {
  const displayPlayers = players.map(player => {
    return (
      <TopPlayer 
        stat={player[stat]}
        statName={stat}
        team={player.team_name}
        name={player.name_display_first_last}
        id={player.player_id}
      />
    )
  });

  return (
    <>
      {displayPlayers}
    </>
  )
}

export default TopPlayersContainer;
