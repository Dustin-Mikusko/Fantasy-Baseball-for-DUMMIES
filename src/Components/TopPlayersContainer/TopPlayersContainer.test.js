import React from 'react';
import TopPlayersContainer from './TopPlayersContainer';
import { shallow } from 'enzyme';

describe('TopPlayersContainer', () => {
  it('should match snapshot with correct data', () => {
    const mockPlayers = [
      {
        stat: 'hr',
        team: 'Houston Astros',
        name: 'Jose Altuve',
        rank: '1'
      }
    ];
    const wrapper = shallow(<TopPlayersContainer 
      stat='hr'
      players={mockPlayers}
    />);
    
    expect(wrapper).toMatchSnapshot();
  })
})
