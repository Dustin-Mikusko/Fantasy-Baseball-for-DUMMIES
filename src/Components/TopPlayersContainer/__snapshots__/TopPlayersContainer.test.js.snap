// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopPlayersContainer should match snapshot with correct data 1`] = `ShallowWrapper {}`;
